tests/jvm/src/test/scala/scala/meta/tests/semanticdb/ExpectSuite.scala

```diff
@@ -125,6 +117,23 @@ trait ExpectHelpers extends FunSuiteLike {
     sorted
   }
 
+  protected def normalizedSymbols(path: Path): Map[String, s.SymbolInformation] = {
```

Comment:
FYI, I had this method in ExpectHelpers because I was in fact reusing it between two test suites at one point, but then I figured out how to make the other suite redundant.